homeworks/roman.lytvynenko_Lytvynenko18/GamesProject/Style/style_games.css

```diff
@@ -0,0 +1,89 @@
+.about-me__header {
```

Comment:
just `game.css` would be enough, as it placed in styles folder

Reply:
done in 5015c401e868e8132d82c925762abaffb9ef6e48